src/analysis/schwartz_space.lean

```diff
@@ -413,6 +420,80 @@ instance : topological_space.first_countable_topology (𝓢(E, F)) :=
 
 end topology
 
+section fderiv
+
+/-! ### Derivatives of Schwartz functions -/
+
+variables {E F}
+/-- The derivative of a Schwartz function as a Schwartz function with values in the
+continuous linear maps `E→L[ℝ] F`. -/
+@[protected] def fderiv (f : 𝓢(E, F)) : 𝓢(E, E→L[ℝ] F) :=
+{ to_fun := fderiv ℝ f,
+  smooth' :=
+  begin
+    have hf := f.2,
+    rw cont_diff_top_iff_fderiv at hf,
+    exact hf.2,
+  end,
+  decay' :=
+  begin
+    intros k n,
+    cases f.3 k (n+1) with C hC,
+    use C,
+    intros x,
+    rw norm_iterated_fderiv_fderiv,
+    exact hC x,
+  end }
+
+@[simp] lemma fderiv_apply (f : 𝓢(E, F)) (x : E) :
+  f.fderiv x = fderiv ℝ f x := rfl
+
+lemma coe_fderiv (f : 𝓢(E, F)) :
+  (f.fderiv : E → (E →L[ℝ] F)) = fderiv ℝ f := rfl
+
+variables (𝕜)
+variables [is_R_or_C 𝕜] [normed_space 𝕜 F] [smul_comm_class ℝ 𝕜 F]
+
+/-- The derivative on Schwartz space as a linear map. -/
+def fderiv_lm : 𝓢(E, F) →ₗ[𝕜] 𝓢(E, E →L[ℝ] F) :=
+{ to_fun := schwartz_map.fderiv,
+  map_add' := λ f g, ext $ λ _, fderiv_add
+    f.differentiable.differentiable_at
+    g.differentiable.differentiable_at,
+  map_smul' := λ a f, ext $ λ _, fderiv_const_smul f.differentiable.differentiable_at a }
+
+@[simp] lemma fderiv_lm_apply (f : 𝓢(E, F)) (x : E) :
+  fderiv_lm 𝕜 f x = fderiv ℝ f x := rfl
+
+lemma coe_fderiv_lm (f : 𝓢(E, F)) :
+  (fderiv_lm 𝕜 f : E → (E →L[ℝ] F)) = fderiv ℝ f := rfl
+
+/-- The derivative on Schwartz space as a continuous linear map. -/
+def fderiv_clm : 𝓢(E, F) →L[𝕜] 𝓢(E, E →L[ℝ] F) :=
+{ cont :=
+  begin
+    change continuous (fderiv_lm 𝕜 : 𝓢(E, F) →ₗ[𝕜] 𝓢(E, E →L[ℝ] F)),
+    refine seminorm.continuous_from_bounded (schwartz_with_seminorms 𝕜 E F)
+      (schwartz_with_seminorms 𝕜 E (E →L[ℝ] F)) _ _,
+    rintros ⟨k, n⟩,
+    use [{⟨k, n+1⟩}, 1, one_ne_zero],
+    intros f,
+    simp only [schwartz_seminorm_family_apply, seminorm.comp_apply, finset.sup_singleton, one_smul],
+    refine (fderiv_lm 𝕜 f).seminorm_le_bound 𝕜 k n (by positivity) _,
+    intros x,
+    rw [coe_fderiv_lm, norm_iterated_fderiv_fderiv],
+    exact f.le_seminorm 𝕜 k (n+1) x,
+  end,
+  ..fderiv_lm 𝕜 }
```

Comment:
```suggestion
  to_linear_map := fderiv_lm 𝕜 }
```
is better definitionally.

Reply:
What is the difference?

Reply:
Your spelling means `to_linear_map := {.. fderiv_lm 𝕜}`